src/functional/resources/dateRange/expected/reposense_testrepo-Charlie_master/commits.json

```diff
@@ -1676,10 +1676,10 @@
     ]
   },
   "authorFinalContributionMap": {
-    "charlesgoh": 948,
-    "jeffreygohkw": 1849,
-    "Esilocke": 3121,
-    "wangyiming1019": 1769
+    "charlesgoh": 1029,
+    "jeffreygohkw": 2191,
+    "Esilocke": 3408,
+    "wangyiming1019": 2411
```

Comment:
Looks like those flags do make a significant difference. We'll seriously consider bringing them back in a future version.